helpers/generate.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
```

Comment:
This file should renamed to `test/task_helper_functions`, which will make it [automatically](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/concourse/build/developer-tools/build/scripts/task_helper_functions.sh) load without requiring an explicit source.

Reply:
File renamed